policy-report/kube-bench-adapter/pkg/report/write.go

```diff
@@ -0,0 +1,76 @@
+package report
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	policyreport "github.com/mritunjaysharma394/policy-report-prototype/pkg/apis/wgpolicyk8s.io/v1alpha2"
+
+	client "github.com/mritunjaysharma394/policy-report-prototype/pkg/generated/v1alpha2/clientset/versioned"
```

Comment:
remove dependency on `github.com/mritunjaysharma394`

Reply:
Thanks a lot, Jim, have removed the dependency on `github.com/mritunjaysharma394`